Resources/Prototypes/_NF/Shipyard/base.yml

```diff
@@ -23,6 +23,16 @@
   components:
     - type: Transform
 
+- type: entity
+  id: SecurityFrontierOutpost
+  parent:
+    - StandardFrontierOutpost  # Parent off of above
```

Comment:
I think we need to not parent off from outposts, and instead just make this its own base class. Reason being, we need to make sure that Borg laws dont allow for some silly 'rules' while aboard an official NT outpost. Aside from that, it would only inherit 'cargo' as a component, and we are likely going to modify this some more later on to get seperate cargo accounts working.

Reply:
Not sure if this is fully addressed in 12e943e .